L1Trigger/TrackFindingTracklet/interface/TrackletProcessorDisplaced.h

```diff
@@ -0,0 +1,87 @@
+#ifndef L1Trigger_TrackFindingTracklet_interface_TrackletProcessorDisplaced_h
+#define L1Trigger_TrackFindingTracklet_interface_TrackletProcessorDisplaced_h
+
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletCalculatorBase.h"
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletCalculatorDisplaced.h"
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletLUT.h"
+#include "L1Trigger/TrackFindingTracklet/interface/CircularBuffer.h"
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletEngineUnit.h"
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletParametersMemory.h"
+#include "L1Trigger/TrackFindingTracklet/interface/TrackletProjectionsMemory.h"
+
+#include <vector>
+#include <tuple>
+#include <map>
+
+namespace trklet {
+
+  class Settings;
+  class Globals;
+  class MemoryBase;
+  class AllStubsMemory;
+  class AllInnerStubsMemory;
+  class VMStubsTEMemory;
+  class StubPairsMemory;
+
```

Comment:
All CMSSW class header files must have a comment at the top explaining what they do. And a comment explaining any non-obvious member functions.

Reply:
added a comment on the top.